Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -171,14 +170,15 @@ std::vector<u32> GetAAModes(u32 adapter_index)
   // Use temporary device if we don't have one already.
   Common::DynamicLibrary temp_lib;
   ComPtr<ID3D11Device> temp_device = device;
+  D3D_FEATURE_LEVEL temp_feature_level = s_supported_feature_levels.back();
```

Comment:
Is this correct if the device is already created? (when it's accessed on 200).

Reply:
You're right, that one is a mistake. Will fix.